raymon/tests/profiling/test_element.py

```diff
@@ -0,0 +1,54 @@
+from raymon.profiling.extractors.structured import ElementExtractor, generate_components
+from raymon.profiling.components import InputComponent
+from raymon.tests.conftest import load_data
+import pandas as pd
+
+
+def test_element():
+    extractor = ElementExtractor(3)
+    extractor.element = 4
+    assert extractor.element == 4
+
+
+def test_extract(load_data):
```

Comment:
This is not how this extractor is supposed to be used. 
1. It is for structured data, not for images
2. It extracts a single element from a vector

So, to test it, you should pass it a vector or pandas series (test both) and extract an element. For example, from a numpy vector, extract the first, secodn and third elements and check whether that was done correctly. Alternatively (= a different test), extract an element from a pandas series that has in index with strings. e.g.: `ElementExtractor(feature_name)`.